gui/src/mainwindow.cpp

```diff
@@ -0,0 +1,1459 @@
+                ui->localImageFileLineEdit->setStyleSheet("color: red;"
+                                                          "border: 1px solid red;");
+                validJob = false;
+            }
+
+
+            //only validate the bbox if the override checkbox is checked
+            if(ui->bboxOverrideCheckBox->isChecked()) {
+                bboxNorth = ui->bboxNorthLineEdit->text().toStdString();
+                bboxSouth = ui->bboxSouthLineEdit->text().toStdString();
+                bboxEast = ui->bboxEastLineEdit->text().toStdString();
+                bboxWest = ui->bboxWestLineEdit->text().toStdString();
+
+                unique_ptr<dg::deepcore::imagery::GdalImage> image;
+                image = boost::make_unique<dg::deepcore::imagery::GdalImage>(ui->localImageFileLineEdit->text().toStdString());
+
+                cv::Rect imageBbox;
+                imageBbox = cv::Rect{ { 0, 0 }, image->size() };
+
+                std::unique_ptr<cv::Rect2d> bbox_entered = nullptr;
+                bbox_entered = boost::make_unique<cv::Rect2d>(cv::Point2d(stod(bboxWest), stod(bboxSouth)),
+                                                              cv::Point2d(stod(bboxEast), stod(bboxNorth)));
+
+                dg::deepcore::geometry::TransformationChain llToPixel {
+                    image->spatialReference().fromLatLon(),
+                    image->pixelToProj().inverse()
+                };
+
+                std::unique_ptr<dg::deepcore::geometry::Transformation> pixelToLL_ = llToPixel.inverse();
+
+                auto bbox = llToPixel.transformToInt(*bbox_entered);
+                auto intersect = imageBbox & (cv::Rect)bbox;
+                try {
+                    DG_CHECK(intersect.width && intersect.height, "Input image and the provided bounding box do not intersect");
+                    hasValidBbox = true;
+                }
+                catch(dg::deepcore::Error e) {
+                    std::clog << e.what() << std::endl;
+                    error += "Input image and the provided bounding box do not intersect\n\n";
+                    validJob = false;
+                }
+
+                if(hasValidBbox && bbox != intersect) {
+                    auto llIntersect = pixelToLL_->transform(intersect);
+                    std::cout << "Bounding box adjusted" << std::endl;
+                }
+            }
+        }
+
+    }
+
+    //validate that bbox coordinates are in valid range
+    if(ui->bboxOverrideCheckBox->isChecked() || ui->imageSourceComboBox->currentText() != "Local Image File") {
+        double north = ui->bboxNorthLineEdit->text().toDouble();
+        double south = ui->bboxSouthLineEdit->text().toDouble();
+        double east = ui->bboxEastLineEdit->text().toDouble();
+        double west = ui->bboxWestLineEdit->text().toDouble();
+
+        if(abs(north) >= 90.0) {
+            error += "North bounding box coordinate is invalid: must be in range (-90,90)\n\n";
```

Comment:
90 degrees is a valid coordinate. Should be `> 90`, `not >= 90`. Actually it should be `> 90 && !almostEq` to account for precision.